### PR TITLE
feature/sva-354-my-experience-profile-buttons 

### PR DESCRIPTION
**Pull request by DJMF2015**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description
- adds logic to toggle between '_Next_' and '_Continue_' buttons for the MyExperienceContainer and hide '_Skip_' button when '_Next_' Button is pressed.
- 'Continue' button turns solid purple.
- **Note**: Continue button does not navigate anywhere as the subsequent containers are not yet built.
 
## Type of change

Please delete options that are not relevant
 
- [] New Feature (non-breaking change which adds functionality) 

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have removed any unnecessary comments or console logging
- [] I have made corresponding changes to the documentation (if required)
- [x] I have addressed accessibility, if needed
- [x] I have followed best practices, e.g. NativeBase approaches and theming
- [x] I have checked the app in dark mode, if making front-end design changes
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have updated the version numbers in `package.json` files in the [app](DEPLOYMENT.md#app-deployment) and/or [api](DEPLOYMENT.md#api-deployment-on-aws) directories as needed

### CODE REVIEW

File: APP_DEVELOPMENT.md [hunk 1]
```diff
@@ -2,26 +2,40 @@
...
-  - [Dark mode](#dark-mode)
-  - [Reusable containers](#reusable-containers)
-  - [Custom components](#custom-components)
-  - [Icons](#icons)
-- [Images](#images)
-- [Logging errors and crashes](#logging-errors-and-crashes)
-  - [Seeing Bugsnag reports](#seeing-bugsnag-reports)
-  - [Logging errors to Bugsnag during development](#logging-errors-to-bugsnag-during-development)
-- [Performance issues](#performance-issues)
+- [App development](#app-development)
```
Comment (PammyLo) [inline APP_DEVELOPMENT.md:5]: I don't think it needs a link to the App development section as we're already here ;)

File: APP_DEVELOPMENT.md [hunk 2]
```diff
@@ -160,6 +174,31 @@ These are custom components we've developed, in addition to NativeBase readymade
 
 Displays the wide version of the STA logo
 
+#### STA Multi-coloured Matchstick Logo
+
+![STA Matchstick](app/src/Assets/Images/STA%20Matchstick%20Logo.png)
+
+STA Matchstick Logo with seven, rotated multi-coloured matchsticks
```
Comment (PammyLo) [inline APP_DEVELOPMENT.md:181]: I can see where you're coming from but this is actually a logo, not a component and doesn't belong here. I think other developers would be able to find this in the logos as that would probably the place where anyone would look for it.

File: APP_DEVELOPMENT.md [hunk 3]
```diff
@@ -160,6 +174,31 @@ These are custom components we've developed, in addition to NativeBase readymade
...
+
+#### STA Single Matchstick
+
```
Comment (PammyLo) [inline APP_DEVELOPMENT.md:184]: Great additions David! Well done on adding three reusable components, awesome :) 

It looks like the naming convention here is to add the reference to the components here with the name as it is in the code, i.e. StaRibbon, Checkbox, ProgressBar.

File: app/src/Assets/Images/Checkbox Group.jpg
Comment (PammyLo) [inline app/src/Assets/Images/Checkbox Group.jpg:1]: It looks like the other screenshots of reusable components are all in the folder: 
 `/Volunteer-app/screenshots/components`

**Review by PammyLo — CHANGES_REQUESTED** [review]
Great work David! @DJMF2015 

Thanks for adding the reusable components to the Readme file, that's very helpful :) 

The added button looks like the design (although I wonder what the intended effect of this button is?) - I just wanted to add here that the progress bar shouldn't move to the next section until the final 'continue' button is clicked as that is when the user progresses to the next stage of the registration. 

I've added a few other small questions/suggestions in the code.

**Comment by M4RC1N76** [comment]
Hi @DJMF2015  David, I have tested the functionality of the 'Next' button in the 'My Experience' section. It works great!! After I have selected Web Developer from the list I am taken to the next screen when I can see 'Done' button (in solid purple). The 'Skip' button is gone by then. I can't seem to find 'Continue' button. on neither of the screens, could you advise on that, please? I am using my Samsung (physical) phone.

**Comment by DJMF2015** [comment]
Hi @M4RC1N76 . It should display 'Done' and not 'Continue' when you see press the button as Joanna wanted me to change this. She also asked me to comment the progress bar out. Does that make sense?

**Comment by M4RC1N76** [comment]
Thanks David, that make sense now. Everything works great!!

**Review by M4RC1N76 — APPROVED** [review]
Everything works great!!

**Review by PammyLo — APPROVED** [review]

**Pull request merged**